admin/scripts/zvlastni/reporty/stravenky.xtpl

```diff
@@ -1,20 +1,27 @@
 <!-- begin:stravenky -->
 <style>
 
-.radek { border: solid 0.3mm; page-break-inside: avoid; display: inline-block; font-family: sans-serif; border-left: none; }
-.radek > div { float: left; width: 16mm; height: 16mm; padding: 2mm; border-left: solid 0.3mm; overflow: hidden; position: relative; }
+.radek { page-break-inside: avoid; display: inline-block; font-family: sans-serif; border-left: none; }
+.radek > div { float: left; width: 22mm; height: 22mm; padding: 2mm; box-shadow:
+    1px 0 0 0,
+    0 1px 0 0,
+    1px 1px 0 0,   /* Just to fix the corner */
+    1px 0 0 0 inset,
+    0 1px 0 0 inset; overflow: hidden; position: relative; }
 .radek > hr { margin: 0; border: 0; clear: both; height: 0; }
-.jidlo > img { width: 100%; position: absolute; top: 0; left: 0; opacity: 0.5; }
+.jidlo { }
+.jidlo > img{ height: 100%; position: absolute; top: 0; right: 0; opacity: 0.5; }
```

Comment:
Nedekódoval jsem, v čem je rozdíl oproti původní variantě. Pokud tam není žádný spec. důvod, vrátil bych to do původního stavu.